src/main/java/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+import core.basesyntax.Figure;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURES_NUMBER = 5;
+    public static final int RANDOM_NUMBER = 30;
+    public static final Random random = new Random();
+    public static final ColourSupplier colourSupplier = new ColourSupplier();
```

Comment:
```suggestion
    private Random random = new Random();
    private ColourSupplier colourSupplier = new ColourSupplier();
```
these fields are not constants